### ISSUE TITLE
"Share this" buttons do not load Facebook and Twitter

### ISSUE THREAD

**Issue by gerdneuman** [mozilla/blok#218]
**Steps to reproduce:**
1. Go to e.g. https://hacks.mozilla.org/2016/11/mozilla-push-server-now-supports-topics/
2. Click on "Share this". This loads G+, but does not load Twitter or Facebook Share buttons:

![_not_2016-11-17 10_55_32-mozilla push server now supports topics mozilla hacks the web developer blog](https://cloud.githubusercontent.com/assets/12525563/20392295/74596cae-acd7-11e6-8bda-ecc629226d89.png)

**If I disable blok on this page, then it works:**

![ok_2016-11-17 10_57_27-mozilla push server now supports topics mozilla hacks the web developer blog](https://cloud.githubusercontent.com/assets/12525563/20392286/6e6d0918-acd7-11e6-9b12-d52c268bac14.png)

**Comment by groovecoder** [comment]
Thanks for the report. Did you also report this thru the add-on UI? That's where we're looking at most of the breakage data.

**Comment by gerdneuman** [comment]
@groovecoder No, only via github. I first tried to report thru the add-on UI, but it did not let me attach a screenshot which I thought would be useful here (And I think it is as you can see above). If you want I could still report the breakage this way and mention this report in there?

**Comment by groovecoder** [comment]
Yes, please go ahead and report via the add-on UI too.

**Comment by gerdneuman** [comment]
ok, I reported it today via the add-on UI.
